pyproject.toml

```
@@ -0,0 +1,10 @@
+[tool.black]
+line-length = 100
```

```suggestion
line-length = 100
skip-string-normalization = 1
```

I checked that this is the correct key